components/rmrk/Gallery/GalleryItem.vue

```diff
@@ -105,7 +105,7 @@
                     </p>
                   </div>
 
-                  <Sharing class="mb-4" />
+                  <Sharing show-download-icon class="mb-4" />
```

Comment:
```suggestion
                  <Sharing enableDownload class="mb-4" />
```
according to `STYLE_GUIDE.md`

```
<MyCustomComponent :propertyName="doSomething">
    {{ someProperty }}
</MyCustomComponent>
```